src/main/java/edu/ucsb/mapache/google/SearchResult.java

```diff
@@ -99,6 +49,35 @@ public static SearchResult fromJSON(String json) {
             logger.error("JsonProcessingException:" + jpe);
             return null;
         }
-        
+
+    }
+
+    /**
+     * Create a SearchResult object from json representation
+     * 
+     * @param json String of json returned by Google Search API
+     * @return a new SearchResult object
+     */
+    public String toJSON() throws JsonProcessingException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        // try {
+        return objectMapper.writeValueAsString(this);
+        // } catch (JsonProcessingException jpe) {
```

Comment:
Should this commented try/catch be removed?

Reply:
Good catch... done!